clients/java/src/main/java/io/zeebe/client/impl/worker/JobWorkerImpl.java

```diff
@@ -45,6 +45,15 @@
   private final AtomicBoolean acquiringJobs = new AtomicBoolean(true);
   private final AtomicReference<JobPoller> jobPoller;
 
+  public JobWorkerImpl(
+      final int maxJobsActive,
+      final ScheduledExecutorService executor,
+      final Duration pollInterval,
+      final JobRunnableFactory jobRunnableFactory,
+      final JobPoller jobPoller) {
+    this(maxJobsActive, 0.3f, executor, pollInterval, jobRunnableFactory, jobPoller);
```

Comment:
Sorry, I think you misunderstood me. Please replace this magic number with `ZeebeClientBuilderImpl.DEFAULT_JOB_WORKER_MIN_JOBS_ACTIVE_RATIO`, which you can define and reuse inside `ZeebeClientBuilderImpl` as `public static final float DEFAULT_JOB_WORKER_MIN_JOBS_ACTIVE_RATIO = 0.3f;`